pipeline/metadata/main.yaml

```diff
@@ -0,0 +1,149 @@
+#=====================================================================================
+# Metadata file for quincy upstream testing.
+# Single file to specify test suites to be executed for all regression (sanity) and schedule tests
+# as per defined in each stages.
+# Each stage will execute in sequential pattern.
+# Contains all Default parameter used for Execution when no override is specified.
+# parameter required to create a similar custom file is suite name, suite yaml file, global configuration file,
+# platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
+#=====================================================================================
+suites:
+- name: "Core Feature Upstream Testing For DMFG"
```

Comment:
Hi Sunil
Can we follow new way of representing metadata.yaml. If it is not time taking

Regards,
Amarnath

Reply:
will take it in next PR.